Sources/docc/DocCDocumentation.docc/other-formatting-options.md

```diff
@@ -0,0 +1,108 @@
+# Other Formatting Options
+
+Improve the presentation and structure of your content by incorporating special page elements.
+
+## Overview
+
+DocC offers support for various types of elements such as tables, notes, asides, and special characters.
+
+### Add Tables of Data
+
+To create a table, start a new paragraph and use dashes (`-`) to define 
+columns, and pipes (`|`) as column separators. Construction of a table requires 
+at least three rows: 
+
+* A row that contains the column names.
+* A row with cells that contain only dashes (`-`); three or more for each 
+column.
+* One or more rows of content.
+
+```markdown
+| Sloth speed  | Description                           |
+| ------------ | ------------------------------------- | 
+| `slow`       | Moves slightly faster than a snail.   |
+| `medium`     | Moves at an average speed.            |
+| `fast`       | Moves faster than a hare.             |
+| `supersonic` | Moves faster than the speed of sound. |
+```
+There's no need to impose a column width. A column determines its width 
+using the contents of its widest cell. You can also omit the leading 
+and trailing pipes.
```

Comment:
Should we also add that they can omit any additional spaces used to "align" the contents of a table within the cell?

Reply:
Added in https://github.com/apple/swift-docc/pull/742/commits/0d0651d7a625986ec00e5b29ecdeae51053d2109